static/js/zamboni/buttons.js

```diff
@@ -17,6 +17,13 @@ var notavail = '<div class="extra"><span class="notavail">{0}</span></div>',
     noappsupport = '<div class="extra"><span class="notsupported">{0}</span></div>',
     download_re = new RegExp('(/downloads/(?:latest|file)/\\d+)');
 
+// Restyle is enabled; we're going to modify the text.
+if ($('body').hasClass('restyle')) {
+    notavail = '<div class="extra"><button class="button disabled concealed not-available">{0}</button></div>';
+    incompat = '<div class="extra"><button class="button disabled concealed not-available">{0}</button></div>';
+    noappsupport = '<div class="extra"><button class="button disabled concealed not-available">{0}</button></div>';
```

Comment:
`.install-button .concealed` is `display: none`, but `.extras .concealed` is not. I could clean up those names if that's too confusing, which it obviously is. :smile: